### DISCUSSION TITLE
How to build sniffer in IROS 2022 Decentralized swarm demo

### DISCUSSION THREAD

**Discussion by rkdals03** [bitcraze/discussions#1069] [Q&A]
Hello! bitcraze Everyone, I tried to run iros 2022 demo to implement decentralized swarm using your product. However, I have not been able to build sniffer. Other drones are built, but only sniffer can not build.

I'm going to use the uri of sniffer as radio:/0/80/2M/E7E7E700
I specified it like this. But I did make load like in the picture but my computer can't find that drone. Can you tell me what to do specifically about this?

Thank you for reading it.

![Screenshot 2023-12-03 at 17 18 53](https://github.com/bitcraze/discussions/assets/65216898/35d21bed-3977-491e-849b-388199fd3fbc)

[iros-2022 decentralized swarm demo](https://github.com/bitcraze/crazyflie-firmware-experimental/tree/iros-2022/examples/demos/decentralized_swarm)

**Comment by krichardsson** [comment]
Hi!

It sounds like you have managed to build the sniffer firmware but that the flashing is not working?

Flashing the sniffer should not be anything special. All that should be required is to put the Crazyflie in booloader mode and then run `make cload`. In this style of flashing the address does not matter as you manually put the Crazyflie in bootloader mode.

Another option is to use the `CLOAD_CMDS` environment variable. **Without** putting the Crazyflie in booloader mode, run `CLOAD_CMDS=`-w radio://0/30/2M/E7E7E7E700` make cload. In this style of flashing the radio tells the Crazyflie to enter booloader mode and the address must match.

By the way, it looks like your URI is not correct, the address part should be longer, perhaps your intention was to use `radio:/0/80/2M/E7E7E7E700`?
